src/libraries/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -871,6 +871,22 @@ public void ReadArchive_WithDiskStartNumberGreaterThanIntMax()
             Assert.Null(exception);
         }
 
+        [Theory]
+        [InlineData("HuffmanTreeException.zip")]
+        public static async Task ZipArchive_InvalidHuffmanData(string zipname)
+        {
+            string filename = bad(zipname);
```

Comment:
nit, this doesn't need to be a Theory (unless you think we'll add many more test files), also add a comment
```suggestion
        /// <summary>
        /// This test checks that an InvalidDataException will be thrown when consuming a zip with bad Huffman data.
        /// </summary>
        [Fact]
        public static async Task ZipArchive_InvalidHuffmanData()
        {
            string filename = bad("HuffmanTreeException.zip");
```